src/Raven.Client/Documents/Operations/ETL/Queue/QueueConnectionString.cs

```diff
@@ -31,6 +33,12 @@ protected override void ValidateImpl(ref List<string> errors)
                     errors.Add($"{nameof(RabbitMqConnectionSettings)} has no valid setting.");
                 }
                 break;
+            case QueueBrokerType.AzureQueueStorage:
+                if (AzureQueueStorageConnectionSettings?.Authentication == null)
```

Comment:
Just checking if it's not null is enough? I think we should rather verify that what was provided there is correct. For example that there is just _one_ auth option provided and that is has all fields / properties filled correctly